Firestore/core/test/unit/local/document_overlay_cache_test.cc

```diff
@@ -152,6 +152,27 @@ TEST_P(DocumentOverlayCacheTest, CanReadSavedOverlays) {
   });
 }
 
+// Ensure that the fix in https://github.com/firebase/firebase-ios-sdk/pull/9381
```

Comment:
I do not think this comment helps here. It is more helpful to explain what this test does, if you think the name is not obvious.

Reply:
Ok. I'll just remove the comment (here and below).